test/integration/targets/ecs_ecr/tasks/main.yml

```diff
@@ -329,6 +329,39 @@
         that:
           - result is not changed
 
+
+    - name: When creating with force_delete flag true
+      ecs_ecr:
+        name: '{{ ecr_name }}'
+        region: '{{ ec2_region }}'
+        force_delete: true
+        ec2_access_key: '{{ec2_access_key}}'
+        ec2_secret_key: '{{ec2_secret_key}}'
+        security_token: '{{security_token}}'
+      register: result
+
+    - name: it should ignored force parameter and preform change and create
+      assert:
+        that:
+          - result is changed
+          - result.created
+
+    - name: When deleting a repository with force_delete flag true
```

Comment:
Please also add a a `check_mode: yes` version of this test and a test that retrying the deletion after the deletion returns `result is not changed`:

```
- name: When deleting a repository with force_delete flag true (check_mode)
  check_mode: yes
  ecs_ecr:
    state: absent
...
- name: it should skip, flag as changed and not delete
  assert:
...
- name: When deleting a repository with force_delete flag true
  check_mode: yes
  ecs_ecr:
    state: absent
...
- name: it should flag as changed and delete
  assert:
...
- name: When deleting a repository with force_delete flag true (test idempotency)
  check_mode: yes
  ecs_ecr:
    state: absent
...
- name: it should not change
  assert:
```